src/library_async.js

```diff
@@ -649,27 +731,39 @@ mergeInto(LibraryManager.library, {
     },
 
     allocateData: function() {
-      // An asyncify data structure has two fields: the
-      // current stack pos, and the max pos.
-      var ptr = _malloc(Asyncify.StackSize + 8);
-      HEAP32[ptr >> 2] = ptr + 8;
-      HEAP32[ptr + 4 >> 2] = ptr + 8 + Asyncify.StackSize;
+      // An asyncify data structure has three fields:
```

Comment:
Perhaps reference the new `asyncify_data_t` struct here?

Reply:
Yeah, that's a good idea. Will do when I get to updating the docs.